primap2/_merge.py

```diff
@@ -54,8 +54,10 @@ def merge_with_tolerance_core(
     da_merge: xr.DataArray
         data to merge into ds_start
     tolerance: float (optional), default = 0.01
-        The tolerance to use when comparing data
-    error_on_discrepancy: (optional), default = True
+        The tolerance to use when comparing data. Tolerance is relative to values in
+        the calling Dataset. Thus by default a 1% deviation of values in da_merge
+        from the calling Dataset is tolerated.
+    error_on_discrepancy (optional), default = True
```

Comment:
Hm, this doesn't seem to have anything to do with data reading? Is this some merge conflict with #118 ?

Reply:
Yes, that edit was due to a merge conflict. I don't know why it shows up here, because I thought I had changed it to the head version from my local version.

Reply:
probably some weirdness, anyway, whatever is new in your commit is clearly better than what was before, so let's take that. :slightly_smiling_face: